tests/Unit/API/Site/Controllers/MerchantCenter/TargetAudienceControllerTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
```

Comment:
Same as in https://github.com/woocommerce/google-listings-and-ads/pull/2003/files#r1245074785